### ISSUE TITLE
IAsyncLogEventSinkInspector inspector.Count is always showing zero

### ISSUE THREAD

**Issue by PrasKulkarni** [serilog/serilog-sinks-async#84]
I have configured AmazonKinesis as my sink, when I do health check I always get zero for inspector.Count even after writing logs. Am I missing anything here? 

Below is the code I am using:

public class MonitorConfiguration : IAsyncLogEventSinkMonitor
        {
            IAsyncLogEventSinkInspector inspector;
            public IAsyncLogEventSinkInspector Inspector => inspector;
            public void StartMonitoring(IAsyncLogEventSinkInspector inspector)
            {
                this.inspector = inspector;
               
            }

            public void StopMonitoring(IAsyncLogEventSinkInspector inspector)
            { /* reverse of StartMonitoring */ }
        }

config.WriteTo.Async((s => s.AmazonKinesis(options, kinesisClient)), monitor: monitor);

**Comment by bartelink** [comment]
For issues like this, it's recommended to ask on StackOverflow.com, tagging it as `serilog`. That way you'll get a broad mix of people who understand Serilog, and the specific sink in question.

Only if you've determined that the Async Sink is the culprit, and have a clear repo, does it make sense to log it here and get the few watchers this repo has to pay specific attention.

In this instance, my money is on removing `Async` not changing things. Sinks in general tend to buffer the events, so the `Async` backlog will only rarely get long, and this is not intended as a primary diagnostic tool as you are using it. That collapses your question to "how do I troubleshoot the `AmazonKinesis` Sink", which is a very specific question that I would not expect a random person looking at this repo to know (all it makes me think is "of all the ways to manage logs in AWS, why Kinesis?")

So, in short, I'd close this and create a SO question with what you've tried, and your full wireup code (excluding Async bits) - if you tag it `serilog`, >60% of the people watching this repo will see it there too (and you'll get to tag it `amazon-kinesis` and get insight from people watching that tag)

Finally, triple tick (`) symbols around code make things more legible (here and on SO)
